### PR TITLE
VRF:FluidTemperatureControl with Fan:systemModel runs continuously in fan cycling mode 

### LINKED ISSUE #7066: VRF:FluidTemperatureControl with Fan:systemModel runs continuously in OnOff operating mode
Issue overview
--------------
Same issue as #6336, but this time for VRF:FluitTemperatureControl. Issue was fixed for the other VRF in #6342. Need similar code changes in `VRFTerminalUnitEquipment::CalcVRF_FluidTCtrl`.

### Details
Some additional details for this issue (if relevant):
 - Platform (Operating system, version)
 - Version of EnergyPlus (if using an intermediate build, include SHA)
 - Helpdesk ticket number 12348.

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Defect file added to EnergyPlusDevSupport\DefectFiles
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### PR DESCRIPTION
**Pull request by Nigusse**
Pull request overview
---------------------
Fixes #7066. After the fix the Fan:SytemModel fan is off whenever there is NO cooling/heating load and the fan is in cycling mode. Diffs are expected if Fan:SytemModel is used in VRF air terminal unit with Fluid Control model and the fan is set to run in cycling operating mode.

### Work Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] At least one of the following appropriate labels must be added to this PR to be consumed into the changelog:
   - Defect: This pull request repairs a github defect issue.  The github issue should be referenced in the PR description
   - Refactoring: This pull request includes code changes that don't change the functionality of the program, just perform refactoring

### Review Checklist
This will not be exhaustively relevant to every PR.
 - [ ] Functional code review (it has to work!)
 - [ ] If defect, results of running current develop vs this branch should exhibit the fix
 - [ ] CI status: all green or justified
 - [ ] Performance: CI Linux results include performance check
 - [ ] Unit Test(s)
 - C++ checks:
   - [ ] Argument types
   - [ ] If any virtual classes, ensure virtual destructor included, other things
 - [ ] If new idf included, locally check the err file and other outputs
 - [ ] Required documentation updates?
 - [ ] If output changes, including tabular output structure, add to output rules file for interfaces

### CODE REVIEW

**Comment by Nigusse** [comment]
I created defect files to confirm the issue. I could not compare the Fan:SytemModel fan runs against Fan:OnOff model run since the later is not supported in VRF air terminal unit with Fluid Control. However, I was able to compare against a modified test with Fan:VariableVolume just to verify consistency of trend. After the fix the Fan:SytemModel fan is off whenever there is NO cooling/heating load and the fan operating mode is cycling.

**Comment by Nigusse** [comment]
Sample result for VRF Fluid Control with Fan:SytemModel fan on cycling operating mode before and after the fix. The after the fix result show that the fan is off when there is no cooling/heating load whereas before the fix result show the fan runs continuously at full flow when there is no load.

VRF_wFluidControl_sysfan_OnOFF

**Comment by Nigusse** [comment]
added defect files.

**Comment by Nigusse** [comment]
This branch is ready for review.

**Comment by nrel-bot-3** [comment]
@Nigusse @lgentile it has been 28 days since this pull request was last updated.

**Comment by Nigusse** [comment]
Merged in develop to updated the branch. This is ready for review. @Myoldmopar

File: tst/EnergyPlus/unit/HVACVariableRefrigerantFlow.unit.cc [hunk 1]
```diff
@@ -7604,4 +7608,2316 @@ TEST_F(EnergyPlusFixture, VRFTU_SupplementalHeatingCoilCapacityLimitTest)
...
+        "     Enthalpy,                !- Fluid Property Type",
+        "     R410aSuperHeatTemperatures,  !- Temperature Values Name",
+        "     1.3860E+05,              !- Pressure {Pa}",
+        "     0.0000E+00,0.0000E+00,0.0000E+00,0.0000E+00,0.0000E+00,4.0260E+05,4.0510E+05,",
+        "     4.0757E+05,4.1002E+05,4.1244E+05,4.1485E+05,4.1726E+05,4.1965E+05,4.2204E+05,",
+        "     4.2444E+05,4.2683E+05,4.2922E+05,4.3162E+05,4.3402E+05,4.3642E+05,4.3883E+05,",
+        "     4.4125E+05,4.4368E+05,4.4611E+05,4.4855E+05,4.5100E+05,4.5346E+05,4.5593E+05,",
+        "     4.5841E+05,4.6090E+05,4.6340E+05,4.6591E+05,4.6843E+05,4.7097E+05,4.7351E+05,",
+        "     4.7606E+05,4.7863E+05,4.8121E+05,4.8380E+05,4.8640E+05,4.8902E+05,4.9165E+05,",
+        "     4.9428E+05,4
```
Comment (Myoldmopar) [inline tst/EnergyPlus/unit/HVACVariableRefrigerantFlow.unit.cc]: I think this is going to fail with other recent code changes.  I'll pull in develop locally and confirm and fix it.

File: src/EnergyPlus/HVACVariableRefrigerantFlow.cc [hunk 2]
```diff
@@ -11029,7 +11029,7 @@ namespace HVACVariableRefrigerantFlow {
                 if (OnOffAirFlowRatio > 0.0) {
                     HVACFan::fanObjs[VRFTU(VRFTUNum).FanIndex]->simulate(1.0 / OnOffAirFlowRatio, ZoneCompTurnFansOn, ZoneCompTurnFansOff, _);
                 } else {
-                    HVACFan::fanObjs[VRFTU(VRFTUNum).FanIndex]->simulate(1.0, ZoneCompTurnFansOn, ZoneCompTurnFansOff, _);
+                    HVACFan::fanObjs[VRFTU(VRFTUNum).FanIndex]->simulate(PartLoadRatio, ZoneCompTurnFansOn, ZoneCompTurnFansOff, _);
```
Comment (Myoldmopar) [inline src/EnergyPlus/HVACVariableRefrigerantFlow.cc:11032]: Yeah, this makes good sense.

**Review by Myoldmopar — APPROVED** [review]

**Comment by Myoldmopar** [comment]
Results look great, I pulled the branch down, fixed up the unit test build issue and the unit tests were running fine.  CI is happy.  Merging this in.

**Pull request merged**